_includes/modern-appendix.md

```diff
@@ -413,6 +413,29 @@ Sent as a reply to the {% message AWAY %} command, this lets the client know tha
 
 Sent as a reply to the {% message AWAY %} command, this lets the client know that they are set as being away. The text used in the last param of this message may vary.
 
+{% numericheader RPL_WHOREPLY %}
+
+      "<client> <channel> <username> <host> <server> <nick> <flags> :<hopcount> <realname>"
+
+Sent as a reply to the {% message WHO %} command, this numeric gives information about the client with the nickname `<nick>`. `<username>` and `<realname>` represent the names set by the {% message USER %} command (though `<username>` may be set by the server in other ways). `<host>` represents the host used for the client in nickmasks (which may or may not be a real hostname or IP address). `<host>` CANNOT start with a colon `(':', 0x3A)` as this would get parsed as a trailing parameter – IPv6 addresses such as `"::1"` are prefixed with a zero `('0', 0x30)` to ensure this. `<server>` is the name of the server the client is connected to. `<channel>` is an arbitrary channel the client is joined to or a literal asterisk character `('*', 0x2A)` if no channel is returned.
```

Comment:
It might be best to refer to RPL_WHOISUSER instead of copying it here, so they don't go out of sync with future updates.